.github/workflows/test-release.yml

```diff
@@ -0,0 +1,64 @@
+name: Release
+
+# release branches (and nightly tags) will get the full run across all OS/browsers
+# for multiple node versions
+
+on:
+  push:
+    branches:
+      - release-*
+    tags:
+      - "v0.0.0-nightly-*"
+    paths-ignore:
+      - "docs/**"
+      - "scripts/**"
+      - "**/README.md"
+
+jobs:
```

Comment:
Currently this is the same as `test-full.yml` but using node 18+20 instead of latest